PowerUp/app/src/main/AndroidManifest.xml

```diff
@@ -40,14 +40,27 @@
 
         <activity
             android:name=".DressingRoomActivity"
+<<<<<<< HEAD
```

Comment:
Please remove these merge conflict messages,these won't let the code run properly